### PR TITLE
Add CSS to make angular material work correctly. Centered popup, adde…

### PR DESCRIPTION
**Pull request by kstarrmitche**
…d cdk-overlay, and "no valid data" when user tries to parse markdown when userInput is empty.

### CODE REVIEW

**Comment by kstarrmitche** [comment]
image

File: src/app/popup/popup.component.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
+<h1 mat-dialog-title>Export as HTML...</h1>
 <div mat-dialog-content>
-  <p>{{data.convertedHTML}}</p>
+    <pre class="prettyprint" *ngIf="data.convertedHTML !== '' ; else noValidMarkdown"><code class="language-html">{{data.convertedHTML}}</code></pre>
```
Comment (perrywarner) [inline src/app/popup/popup.component.html:3]: Kathy says this specific library doesn't do exactly what's necessary, but it'll remain for now.

File: src/app/popup/popup.component.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import { Component, OnInit, Inject, OnChanges, Input } from '@angular/core';
+import { Component, Inject, OnChanges, Input } from '@angular/core';
 import {MatDialog, MatDialogRef, MAT_DIALOG_DATA} from '@angular/material';
-
+import * as PR from 'google-code-prettify/src/prettify.js';
```
Comment (perrywarner) [inline src/app/popup/popup.component.ts:3]: import * from all huh

**Review by perrywarner — APPROVED** [review]
OK. check comments

**Comment by perrywarner** [comment]
[Place popup.component in a well-designed location](https://trello.com/c/eg2hEeX0/70-place-popupcomponent-in-a-well-designed-location)

**Pull request merged**